### DISCUSSION TITLE
Is it available to use senmantic-release in cmake-based c++ project ? Thx.

### DISCUSSION THREAD

**Discussion by Kiddinglife** [semantic-release/semantic-release#2110] [Q&A] [answered]
In other words, does semantic-release only works with nodejs project ?

**Comment by travi** [comment, marked as answer]
does this section from our FAQ answer your question? https://semantic-release.gitbook.io/semantic-release/support/faq#can-i-use-semantic-release-to-publish-non-javascript-packages
